constructor/utils.py

```diff
@@ -149,8 +152,8 @@ def get_final_url(info, url):
         if url.startswith(src):
             new_url = url.replace(src, dst)
             if url.endswith(".tar.bz2"):
-                print("WARNING: You need to make the package {} available "
-                      "at {}".format(url.rsplit('/', 1)[1], new_url))
+                logger.warning("You need to make the package {} available "
```

Comment:
%-format

Reply:
Updated the PR description with some reasoning on this topic.